api/DpiAware.md

```diff
@@ -21,9 +23,6 @@ Table of contents:
 
 ## Introduction
 
-By default if DPI awareness is not enabled in application, then OS performs display scaling. That causes text to look blurry on high DPI displays. To resolve this you have to
- call `cef.DpiAware.EnableHighDpiSupport` method. High DPI support is available only on Windows.
-
 Enabling High DPI support in app can be done by embedding a DPI awareness xml manifest in both main executable and subprocess executable (see [Issue #112](../issues/112) comment #2), or by calling the `cef.DpiAware.EnableHighDpiSupport` method.
```

Comment:
Add information that it is recommended to embed a DPI awareness manifeest in subprocess executable. If it's not embedded Chromium will set DPI awareness for processes during runtime, but it's not always reliable, so it's best to embed a manifest.